### PR TITLE
Create Button disabled

### PR DESCRIPTION
**Pull request by nikita-ng**

### CODE REVIEW

File: ui/src/scenes/Projects/components/ProjectCreate/index.js [hunk 1]
```diff
@@ -31,7 +36,20 @@ class ProjectCreate extends Component {
       }
     );
   };
-
+ 
+  onFormChange(e) {
+    this.setState({
+      [e.target.name]: e.target.value
+    },function() {
+      if(this.state.name && this.state.description && this.state.price && this.state.targetDelivery && this.state.spec) {
```
Comment (liorsaar1) [inline ui/src/scenes/Projects/components/ProjectCreate/index.js]: 1. please add comments

**Review by liorsaar1 — CHANGES_REQUESTED** [review]
1. please add comments
2. check for empty fields (null, white spaces)

**Comment by nikita-ng** [comment]
Please review the code. Changes has been done as said.

File: ui/src/scenes/Projects/components/ProjectCreate/index.js [hunk 2]
```diff
@@ -31,7 +37,40 @@ class ProjectCreate extends Component {
...
+  onFormChange(e) {
+
+    //Setting the required states with change of form field
+    this.setState({
+      [e.target.name]: e.target.value
+    },function() {
 
+      //Checking the fields to be null
+      if( this.state.name && 
+        this.state.description &&
```
Comment (liorsaar1) [inline ui/src/scenes/Projects/components/ProjectCreate/index.js]: please refactor.

this.enable = isFormValid(this.state);

```
function isFormValid(state) {
  if isEmpty(state.name) return false;
  if isEmpty(state.description) return false;
  ...
  ...
}
```

**Review by liorsaar1 — CHANGES_REQUESTED** [review]

**Comment by nikita-ng** [comment]
Refactoring done.

File: ui/src/scenes/Projects/components/ProjectCreate/index.js [hunk 3]
```diff
@@ -31,7 +37,40 @@ class ProjectCreate extends Component {
...
+      return false;
+    if(this.isEmpty(state.spec) || this.isEmpty(state.spec.trim())) 
+      return false;
+    return true;
+  }
 
+  //Checking the empty string
+  isEmpty(str) {
+    return (!str || 0 === str.length);
+  }
```
Comment (liorsaar1) [inline ui/src/scenes/Projects/components/ProjectCreate/index.js]: state.name.trim()  should be inside isEmpty()

**Review by liorsaar1 — CHANGES_REQUESTED** [review]

**Comment by nikita-ng** [comment]
Updation in isEmpty().

**Review by liorsaar1 — APPROVED** [review]

**Pull request merged**